osu.Game/Screens/Play/ReplaySettings/PlaybackSettings.cs

```diff
@@ -13,19 +16,43 @@ public class PlaybackSettings : ReplayGroup
         public IAdjustableClock AdjustableClock { set; get; }
 
         private readonly ReplaySliderBar<double> sliderbar;
+        private readonly OsuSpriteText multiplierText;
 
         public PlaybackSettings()
         {
-            Child = sliderbar = new ReplaySliderBar<double>
+            Children = new Drawable[]
             {
-                LabelText = "Playback speed",
-                Bindable = new BindableDouble(1)
+                new Container
                 {
-                    Default = 1,
-                    MinValue = 0.5,
-                    MaxValue = 2,
-                    Precision = 0.01,
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.CentreLeft,
+                            Text = "Playback speed",
+                        },
+                        multiplierText = new OsuSpriteText
+                        {
+                            Anchor = Anchor.CentreRight,
+                            Origin = Anchor.CentreRight,
+                            Text = "1x",
+                            Font = @"Exo2.0-Bold",
+                        }

```

Comment:
How about adding a left/right padding of 20 to these texts? Should align them closer to the design.